### DISCUSSION TITLE
TimeSeriesForestRegressor

### DISCUSSION THREAD

**Discussion by selma-belhadj** [alan-turing-institute/sktime#602] [Q&A]
Hello, I am a little bit confused, I want to use random forest with time series data to predict number of infractions in the future : I found this function TimeSeriesForestRegressor implemented in Sktime but I didn't find any examples I can follow through the examples on Jupyter notebook, and there is not enough documentation on the net, May someone help me or guide me to do so?
Meanwhile I found this part in jupyter notebook examples using RandomForestRegressor of Sklearn and ReducedRegressionForecaster of Sktime: Can someone explain what is the difference between using those 2 functions together and using TimeSeriesForestRegressor?  is it the same role ? !! predicting with time series?

```
regressor = RandomForestRegressor() 
forecaster = ReducedRegressionForecaster(regressor, window_length=10)
```

**Comment by mloning** [comment]
Hi @salam16, 
* TimeSeriesForest is a different algorithm than RandomForest, more precisely it's a modification of RandomForest to handle time series classification problems. Here's the research paper: https://arxiv.org/abs/1302.2277. We adapted the classification algorithm to a regression algorithm by adapting the final decision tree. 
* `ReducedRegressionForecaster` is a reduction approach that lets you apply a regression algorithm to a forecasting problem. We describe how this works in the forecasting example notebook. 

Does that help?
